doc/Hammer-Use/Hammer-APIs.rst

```diff
@@ -33,7 +33,58 @@ Pin constraints are specified using the Hammer IR key ``vlsi.inputs.pin``. PinAs
 
 Power Straps
 ------------
-Power strap constraints are specified using multiple Hammer IR keys in the ``par`` namespace. The currently supported API supports power strap generation by tracks, which auto-calculates power strap width, spacing, set-to-set distance, and offsets based on basic DRC rules specified in the technology Stackup object. The basic pieces of information needed are the desired track utilization per strap and overall power strap density. Different values can be specified on a layer-by-layer basis by appending ``_<layer name>`` to the end of the desired option.
+Power strap constraints are specified using multiple Hammer IR keys in the ``par`` namespace.You can find the keys in <tech>/defaults.yml under the tech plugin directory. An example from asap7 (https://github.com/ucb-bar/hammer/blob/master/hammer/technology/asap7/defaults.yml) is as follows:
```

Comment:
Hyperlinks should be like this:
`[asap7](https://github.com/ucb-bar/hammer/blob/master/hammer/technology/asap7/defaults.yml)`